contracts/modules/capital/Pool.sol

```diff
@@ -77,44 +78,45 @@ contract Pool is IPool, MasterAwareV2, ReentrancyGuard {
     _;
   }
 
+
   constructor (
```

Comment:
the constructor is quite complex now.

It deserves its own unit test at this point

Reply:
Yea I agree on this point. I wanted to update to ethers before writing the tests though, so maybe I will add them to the PR this is merged into?

Reply:
oh yeah sure where it make sense

Reply:
let's do this in the other PR right